src/app/state.rs

```diff
@@ -65,6 +72,16 @@ impl Action {
 }
 
 impl AppState {
+    fn new(cx: Scope) -> Self {
+        tracing::debug!("🔨 Creating AppState default value");
+        let recent_flakes =
+            storage::<LocalStorage, _>(cx, "recent_flakes".to_string(), FlakeUrl::suggestions);
+        AppState {
+            recent_flakes,
+            ..AppState::default()
+        }
+    }
```

Comment:
This (`storage::<LocalStorage, _>`) is how we create a dioxus `Signal` that is backed by a persistent storage behind the scenes, thus making the signal itself persistent across app restarts!